src/CloudWatchLogs.jl

```diff
@@ -38,6 +39,10 @@ const GENERIC_AWS_DELAYS = ExponentialBackOff(n=10, first_delay=0.2, factor=2, j
 
 __init__() = Memento.register(LOGGER)
 
+function logs(config::AWSConfig, operation::AbstractString; kwargs...)
```

Comment:
After merging can we create an issue to call functions properly? I don't mind this to unblock but it just adds in a bit of debt.

Reply:
I could do that as part of this PR, I was just being lazy...

Reply:
Hehe, honestly it's your call if you want to spend the time. I'm fine with this wrapper function here.

Reply:
Replaced